js/commonMenu.js

```diff
@@ -151,7 +151,7 @@ module.exports.preferencesMenuItem = {
 
 module.exports.bookmarksMenuItem = {
   label: 'Bookmarks manager...',
-  accelerator: 'CmdOrCtrl+Alt+b',
+  accelerator: process.platform === 'win32' ? 'Shift+Alt+B' : 'CmdOrCtrl+Alt+B',
```

Comment:
Right idea to fork it but pls use the same shortcut as Chrome here for familiarity's sake.  That's  Control+Shift+O for bookmarks manager on Windows.  And Command+Alt+B for OS X.